hooks/Makefile

```diff
@@ -0,0 +1,6 @@
+# Install git hooks for development.
+install:
+	@cp ./pre-commit.sh ./../.git/hooks/pre-commit
+	@chmod +x ./../.git/hooks/pre-commit
+
+.PHONY: install
```

Comment:
Please check this everytime.. Endlines.. :P

Reply:
I have added an editorconfig, so issues like this won't be there from now on.